sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.22.2 (2024-11-18)
+## 1.22.4 (2024-11-21)
 
 ### Bugs Fixed
-  - **#3620407** - Fix Datastore credentials show up as NoneCredentials
```

Comment:
Are we intentionally removing previous changelogs?

Reply:
there were duplicate entries which I removed.